src/main/java/io/github/swagger2markup/builder/Swagger2MarkupConfigBuilder.java

```diff
@@ -554,6 +562,7 @@ public Swagger2MarkupConfigBuilder withLineSeparator(LineSeparator lineSeparator
         private String securityDocument;
         private String separatedOperationsFolder;
         private String separatedDefinitionsFolder;
+        private Character documentFolderSeparator;
```

Comment:
I made these changes as well and added an additional attribute for explicitly enabling/disabling rather than the existence of the delimiter.